### PR TITLE
Wrote README.md for both replace-important packages

### PR DESCRIPTION
**Pull request by torch2424**
As the title states. Followed guides on writing good READMEs, and added a gif for the replace-important CLI.

# Guides

https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3
https://lizthedeveloper.gitbooks.io/documentation/content/quickstarts.html

### CODE REVIEW

File: tools/gulp-replace-important/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# gulp-replace-important
+
+Gulp plugin for [replace-important](../replace-important). Plugin for removing !important from css rules from selectors and replacing them inside similar selectors - the new selectors will have the highest specificity in the CSS file. This is useful to remove any code smells from !important, and for making CSS [Supported by AMPHTML](https://www.ampproject.org/docs/guides/responsive/style_pages). This plugin only supports Buffers, but stream support could be added with [gulp-streamify](https://github.com/nfroidure/gulp-streamify).
```
Comment (camelburrito) [inline tools/gulp-replace-important/README.md]: can we replace smells with something else?

File: tools/gulp-replace-important/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+npm run test
+```
+
+# Contributing
+
+Please see the [CONTRIBUTING.md](../../CONTRIBUTING.md)
+
+# Credits
+
+Logic for Replace Important done by [@camelburrito](https://github.com/camelburrito) . Node Module Wrapping, Tests, and Gulp Plugin done by [@torch2424](https://github.com/torch2424) .
```
Comment (camelburrito) [inline tools/gulp-replace-important/README.md]: i think we should remove this section

File: tools/replace-important/README.md [hunk 3]
```diff
@@ -1,2 +1,81 @@
...
+
+```
+npm run test
+```
+
+# Contributing
+
+Please see the [CONTRIBUTING.md](../../CONTRIBUTING.md)
+
+# Credits
```
Comment (camelburrito) [inline tools/replace-important/README.md]: lets remove this section

File: tools/gulp-replace-important/README.md (hunk 1 above)
Comment (torch2424) [inline tools/gulp-replace-important/README.md]: Done

File: tools/gulp-replace-important/README.md (hunk 2 above)
Comment (torch2424) [inline tools/gulp-replace-important/README.md]: Done

File: tools/replace-important/README.md (hunk 3 above)
Comment (torch2424) [inline tools/replace-important/README.md]: Done

**Pull request merged**